# Code review

In this assignment, we will provide the information, link, the paper, and the related code that will be used for this project.

## Papers

The sequence to be analyzed will be taken from the research paper [Chrzastek et.al, 2023](https://bioone.org/journals/avian-diseases/volume-66/issue-4/aviandiseases-D-22-99999/A-Universal-Single-Primer-Amplification-Protocol-to-Perform-Whole-Genome/10.1637/aviandiseases-D-22-99999.full). Following that, sequences will be analyzed using the protocol mentioned in [Chrzastek et.al, 2023](https://bioone.org/journals/avian-diseases/volume-66/issue-4/aviandiseases-D-22-99999/A-Universal-Single-Primer-Amplification-Protocol-to-Perform-Whole-Genome/10.1637/aviandiseases-D-22-99999.full) and [Egana-labrin et. al, 2019](https://www.nature.com/articles/s41598-019-45494-4#Sec7). Shortly,the sequence will be aligned to chicken genome (galGal5) using BWA-MEM. From the non-host reads contigs will be made and aligned to identified viral contigs to find the number of viral reads. The obtained gene sequences were compared to the vaccine strain S1133 and sequence homologies will be calculated. 

## Associated code
The code for this script can be found in [script](https://github.com/Nneupane133/Code_review/new/main)
